CONTRIBUTING.md

```diff
@@ -68,7 +68,7 @@ It's possible to take only a subset of the screenshots which greatly speeds up t
 To do this follow the following steps:
 
 
-1. In `Screenshots.swift` change the `runOnly` property per the guide.
+1. In `Example/Backpack Screenshot/BackpackSnapshotTestCase.swift` change the `runOnly` property per the guide in the comment.
```

Comment:
```suggestion
1. In `Example/Backpack Screenshot/Screenshots.swift` set the property `runOnly` in the `setUp` method to the set of components you want to take screenshots of. Like so: `runOnly = Set(["Nudger", "Chips"])`
```

Reply:
`Screenshots.swift` is the wrong file hence the suggestion here, this is what prompted this PR in the first place as its not inside that file when searched but the `BackpackSnapshotTestCase.swift`